packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapController.m

```diff
@@ -259,6 +270,7 @@ - (void)onMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
     if ([markerIdsToRemove isKindOfClass:[NSArray class]]) {
       [self.markersController removeMarkersWithIdentifiers:markerIdsToRemove];
     }
+    [self.clusterManagersController clusterAll];
```

Comment:
I feel like this should be in a separate method channel method. Can there be a situation when some markers are updated and clusterAll is not desired?

Reply:
I don't see use case where developers would updated markers in steps, as current platform interface supports updating all markers in one update-set, and that should be encouraged way. Having this clustering method as separate method channel method would just complicate the API, and developers would need to call clustering separately for each cluster manager instead automatic update. Of course this would give a bit more control for the developers when items should be clustered, but in reality I don't see when this would be needed.